### PR TITLE
Migrate role members screen

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://github.com/gravitee-io/issues/issues/6376

**Description**

 - Migrate role members screen
 - Use default gio badge in OrgSetting user screens

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-ftosflcmih.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6376-migrate-role-members/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-sktxhrlskf.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6376-migrate-role-members/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/services-ngx/role.service.spec.ts [hunk 1]
```diff
@@ -72,6 +72,23 @@ describe('RoleService', () => {
...
+      const username = 'johndoe';
+
+      roleService.deleteMembership(roleScope, roleName, username).subscribe(() => done());
+
+      httpTestingController
+        .expectOne({
+          method: 'DELETE',
+          url: `${CONSTANTS_TESTING.org.baseURL}/configuration/rolescopes/${roleScope}/roles/${roleName}/users/${username}`,
+        })
+        .flush(null);
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/services-ngx/role.service.spec.ts:88]: no more `{status: 204}` ?

Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/services-ngx/role.service.spec.ts:88]: do you prefer to use null for the void now?

File: gravitee-apim-console-webui/src/organization/configuration/roles/org-settings-role-members.component.ts [hunk 2]
```diff
@@ -72,6 +77,29 @@ export class OrgSettingsRoleMembersComponent implements OnInit, OnDestroy {
...
   onAddMemberClicked() {}
 
-  // eslint-disable-next-line @typescript-eslint/no-empty-function,@typescript-eslint/no-unused-vars
-  onDeleteMemberClicked(member) {}
+  onDeleteMemberClicked(membership: MembershipListItem) {
+    this.matDialog
+      .open<GioConfirmDialogComponent, GioConfirmDialogData, boolean>(GioConfirmDialogComponent, {
+        width: '500px',
+        data: {
+          title: 'Delete a membership',
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/organization/configuration/roles/org-settings-role-members.component.ts:85]: 🤔  Should we use Delete or Remove ? cci  @ ibahloul

**Review by ThibaudAV — APPROVED** [review]
LGTM 🚀

File: gravitee-apim-console-webui/src/services-ngx/role.service.spec.ts (hunk 1 above)
Comment (gaetanmaisse) [reply gravitee-apim-console-webui/src/services-ngx/role.service.spec.ts:88]: I was wrongly assuming the param was parser and so status was used to set response status but it wasn't, my bad. So I used null instead. As discussed I will update all the `{status: 204}` I put in my previous PRs

File: gravitee-apim-console-webui/src/organization/configuration/roles/org-settings-role-members.component.ts (hunk 2 above)
Comment (gaetanmaisse) [reply gravitee-apim-console-webui/src/organization/configuration/roles/org-settings-role-members.component.ts:85]: Need to be defined and put in the guideline + mockup

**Pull request merged**